_pages/how-we-work/tools/tock.md

```diff
@@ -66,20 +66,25 @@ You must not bill for the following activities because these are indirect costs
 
 ### How many hours am I expected to bill per week?
 
-The majority (80%) of your time at 18F will be dedicated to your billable project work. If it's not explicitly an indirect cost, it's project work.
+If you’re an individual contributor or a lead, you’re expected to spend at least  80% of your time  on billable project work. (Directors and supervisors have different billability expectations.)
 
-To keep projects moving forward as planned in our SOWs, based on the number of full (8-hour) days you work in a given week, you need to spend at least the following number of hours on project work:
+That leaves up to 20% of your time to be spent on non-billable work. There are two types of non-billable work, both of which are used to manage the running of 18F itself: GSA-mandated non-billable work and 18F non-billable work. 
 
-Days in the office for the week | Hour Expectation
---- | ---
-5 days | At least 32\* billable hours<br>Up to 8 hours non-billable
-4 days | At least 28\* billable hours<br>Up to 4 hours non-billable
-3 days | At least 22\* billable hours<br>Up to 2 hours non-billable
-2 days | At least 16\* billable hours<br>Up to 0 hours non-billable
-1 days | At least 8\* billable hours<br>Up to 0 hours non-billable
+GSA-mandated non-billable work includes time spent in HRLinks, mandatory OLU trainings, IDPs, SF-182s, annual reviews, and so forth. These are the things we don’t have a choice but to spend time on.
 
+18F non-billable work includes time spent in guilds and working groups, team coffees, chapter meetings, business development, conference attendance, and anything that contributes to the running of 18F as an organization. 
 
-_\*Leads have slightly different expectations set per Chapter. If you are a lead, please discuss with your Chapter head._
+Not sure whether work you’re doing is billable? If it’s not explicitly an indirect cost, it’s project work.
+
+So for most people in a given 40-hour week, the expectation is to bill at least 32 hours to project work. But what about holidays or leave time? Then you’ll need to spend 80% of the time you’re working that week on billable work. Here’s what that looks like if you’re out of the office for a few days: 
+
+Work week | Billable time (minimum hours expected) | Non-billable time (not to exceed)
+--- | --- | ---
+5 days | 32 hours  (80% of 40 hours) | 8 hours
+4 days | 26 hours (80% of 32 hours) | 6 hours
+3 days | 19.5 hours (80% of 32 hours) | 4.5 hours
+2 days | 13  hours (80% of 16 hours) | 3 hours
+1 days | 6.5 hours (80% of 8 hours) | 1.5 hours
 
 ### What if I don't have enough work to bill to the hour expectation laid out above?
 
```

Comment:
I think it'd be helpful to add something in this section about other places people can pick up some billable hours if their project is light (dev lab micro-requests, etc)

Reply:
I think people should still talk to their supervisor first, especially since the level of billable work across the labs is still pretty low.